package.json

```diff
@@ -69,7 +69,9 @@
     "estraverse": "^4.1.0",
     "exit": "~0.1.2",
     "glob": "^5.0.1",
+    "js-yaml": "~3.4.0",
```

Comment:
Optional deps always install anyway (it's just that installation failure is allowed. read: compilation failure, and js-yaml has no compiled parts). It wouldn't change the installation behavior or requirements.